src/libs/PaymentUtils.js

```diff
@@ -10,7 +10,7 @@ import * as Localize from './Localize';
  * @param {Array} bankAccountList
  * @param {Array} cardList
  * @param {String} [payPalMeUsername='']
- * @returns {Array<PaymentMethod>}
+ * @returns {Array<Object>}
  */
 function getPaymentMethods(bankAccountList, cardList, payPalMeUsername = '') {
```

Comment:
I find it pretty confusing that we have both

`PaymentMethods.getPaymentMethods()` and `PaymentUtils.getPaymentMethods()`

Probably we should rename this to something like `formatPaymentMethods()` since it doesn't actually "get" them and processes a list of things we pass in.